packages/tinymce/README.md

```diff
@@ -0,0 +1,3 @@
+# TinyMCE
+
+Adds a button to WordPress's TinyMCE interface.
```

Comment:
Let's call the package `@web-stories-wp/tinymce-button`. Just `tinymce` is too generic.